skimage/measure/tests/test_regionprops.py

```diff
@@ -299,7 +297,10 @@ def test_centroid_3d():
     cX = Mpqr(0, 0, 1) / Mpqr(0, 0, 0)
     assert_array_almost_equal((cZ, cY, cX), centroid)
 
-    # Test spacing
+def test_spacing_parameter():
+    """Test the _normalize_spacing code."""
```

Comment:
Could I ask you to split these tests into separate functions? Huge test functions are harder to read, debug und update down the line. It's fine to copy a bit of trivial test setup if [pytest's parametrize](https://docs.pytest.org/en/6.2.x/parametrize.html) isn't applicable.

Let us know if you need help with that or want us to take over.

Reply:
I think this is correct.

Reply:
circleCI is failing, but I don't think it is code that I touched. 
https://app.circleci.com/pipelines/github/scikit-image/scikit-image/5681/workflows/32600a76-6966-4220-9f28-26e39c9c9285/jobs/5730